docs/compass/04-01-installation.md

```diff
@@ -339,6 +339,11 @@ Note that using this flag also results in building a new `schema-migrator` image
 ./installation/cmd/run.sh --dump-db --oidc-host {URL_TO_OIDC_SERVER} --oidc-client-id {OIDC_CLIENT_ID} --oidc-admin-group {OIDC_ADMIN_GROUP}
 ```
 
+In order to enable the API Metadata Validator, which is used for validation of ORD Documents (https://github.com/SAP/open-resource-discovery) - execute the installation script with the additional flag that sets the necessary container image of the validator.
```

Comment:
To enable the API Metadata Validator that is used for validating Open Resource Discovery (ORD) documents, run the installation script with an additional flag that sets the required containder image of the validator.